plugins/TapTempo/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+INCLUDE(BuildPlugin)
+
+BUILD_PLUGIN(taptempo TapTempo.cpp MOCFILES TapTempo.h EMBEDDED_RESOURCES artwork.png logo.png)
```

Comment:
`artwork.png` is usually the background image in plugin windows, you could remove it completely.